### PR TITLE
feature: cli command to regenerate files

### PR DESCRIPTION
**Pull request by po1vo**
subj.

### CODE REVIEW

**Comment by stonebuzz** [comment]
Can you create a subdirectory ```command``` in the ```inc``` directory and place the new file there
Best regards

**Comment by stonebuzz** [comment]
For ```Lint```
Can you run this command (at root folder of plugin) 
```shell
../../vendor/bin/licence-headers-check --ansi --no-interaction --directory="." -f
```
this should add the missing header to the new file

**Comment by po1vo** [comment]
@stonebuzz , done.

File: inc/command/regeneratefilescommand.class.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+use Symfony\Component\Console\Command\Command;
+
+class RegenerateFilesCommand extends AbstractCommand
+{
+    protected function configure()
+    {
+        $this->setName('plugin:fields:regenerate_files');
+        $this->setDescription(__('Regenerate container files.', 'fields'));
+        $this->setHelp(
+            __('Check all stored containers files (classes & front) are present and create them if needed', 'fields')
```
Comment (stonebuzz) [inline inc/command/regeneratefilescommand.class.php:43]:
```suggestion
            __('Clean up all files generated by the plugin (classes & front) and  regenerate it', 'fields')
```

**Review by stonebuzz — CHANGES_REQUESTED** [review]

**Review by stonebuzz — APPROVED** [review]

File: inc/command/regeneratefilescommand.class.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+class RegenerateFilesCommand extends AbstractCommand
+{
+    protected function configure()
+    {
+        $this->setName('plugin:fields:regenerate_files');
+        $this->setDescription(__('Regenerate container files.', 'fields'));
+        $this->setHelp(
+            __('Clean up all files generated by the plugin (classes & front) and regenerate it', 'fields')
+            . "\n"
+        );
```
Comment (cedric-anne) [inline inc/command/regeneratefilescommand.class.php:45]:
```suggestion
        $this->setHelp(
            __('This command will clean up all files generated by the plugin and regenerate them.', 'fields')
        );
```

File: inc/command/regeneratefilescommand.class.php [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+use Glpi\Console\AbstractCommand;
+use Symfony\Component\Console\Command\Command;
+
+class RegenerateFilesCommand extends AbstractCommand
+{
+    protected function configure()
+    {
+        $this->setName('plugin:fields:regenerate_files');
+        $this->setDescription(__('Regenerate container files.', 'fields'));
```
Comment (cedric-anne) [inline inc/command/regeneratefilescommand.class.php:41]:
```suggestion
        $this->setDescription(__('Regenerates containers files.', 'fields'));
```

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**